tests/BlockProcessorTest.cpp

```diff
@@ -1,33 +1,39 @@
+// SPDX-FileCopyrightText: 2020-2023 Ravil Dorozhinskii
+//
+// SPDX-License-Identifier: MIT
+
 #include "BlockProcessor.hpp"
-#include "FtyPolicies.hpp"
+#include "FtyDataTypes.hpp"
+#include "FtyExceptions.hpp"
 #include "helper.hpp"
 #include "gtest/gtest.h"
-#include <string>
+#include <iostream>
+#include <iterator>
+#include <list>
 #include <yaml-cpp/yaml.h>
 
 using namespace fty;
 
-bool isEqualBlocks(const BlockT &Left, const BlockT &Right) {
+auto isEqualBlocks(const BlockT& Left, const BlockT& Right) -> bool {
```

Comment:
Why replace the return type with auto and then give it a trailing return type here?